src/plugins/credential_management/public/components/create_button/create_button.tsx

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+// @ts-ignore
```

Comment:
what is this for?

Reply:
bypass type check for that import from json file

Reply:
removed this theme